packages/perseus/src/server-item-renderer.tsx

```diff
@@ -416,6 +416,7 @@ export class ServerItemRenderer
                     content={this.props.item.question.content}
                     widgets={this.props.item.question.widgets}
                     images={this.props.item.question.images}
+                    {...this.props.dependencies}
```

Comment:
**thought**: We also have the ArticleRenderer. Might be good to ensure we're plumbing the dependencies through to the `Renderer` in there. It's not super common to place "answerable" widgets in an Article, but we do have the GradedGroup widget which enables checking answers in an Article and so it's probably good to do so we get accurate analytics from both articles and exercises.

Reply:
Interesting-- that expansion of analytics not related to the current bug, so that should probably be a follow-up ticket for a clean git history.